backend/hypothetical_books_backend/settings.py

```diff
@@ -13,6 +13,7 @@
 from pathlib import Path
 import environ
 import os
+import datetime
```

Comment:
Don't think either of these imports are needed.

Reply:
fixed